controllers/authpolicy_controller_test.go

```diff
@@ -282,6 +272,7 @@ var _ = Describe("AuthPolicy controller", func() {
 			}
 			err = k8sClient.Create(context.Background(), otherRoute)
 			Expect(err).ToNot(HaveOccurred())
+			Eventually(testRouteIsAccepted(client.ObjectKeyFromObject(otherRoute)), time.Minute, 5*time.Second).Should(BeTrue())
```

Comment:
This is fixing https://github.com/Kuadrant/kuadrant-operator/actions/runs/6985513462/job/19021747660?pr=339